content/sdk/00.index.md

```diff
@@ -1,14 +1,31 @@
 ---
 title: "Introduction"
 description: "ZKsync Era SDKs empower developers to seamlessly interact with the ZKsync Era network."
-tags: ["zksync", "sdks", "ZKsync Era SDK", "ethereum", "era network", "javascript", "go", "swift", "python", "java", "rust"]
+tags: ["zksync", "sdks", "zksync Era SDK", "ethereum", "era network", "javascript", "go", "swift", "python", "java", "rust"]
 ---
 
 ## What is ZKsync Era SDKs?
 
-ZKsync Era SDKs help developers create decentralized applications (dApps) for the ZKsync Era network. ZKsync Era is a
-Layer 2 solution for Ethereum. It uses ZK rollup technology to provide scalable and low-cost transactions, keeping the
-security and trust of the Ethereum blockchain.
+**ZKsync Era SDKs** are tools that help developers build decentralized applications (dApps) for the ZKsync Era network.
+
+::div{class="text-center"}
+![ZKsync Era SDKs](/images/sdk/zksync-era-sdks.png){ width="50%"}
+::
```

Comment:
We need to remove this image for now. It is not using the latest branding design.